libsolidity/ast/Types.h

```diff
@@ -1340,7 +1340,9 @@ class FunctionType: public Type
 	bool const m_arbitraryParameters = false;
 	bool const m_gasSet = false; ///< true iff the gas value to be used is on the stack
 	bool const m_valueSet = false; ///< true iff the value to be sent is on the stack
-	bool const m_bound = false; ///< true iff the function is called as arg1.fun(arg2, ..., argn)
+	/// true iff the function is called as arg1.fun(arg2, ..., argn)
+	/// e.g. false for contract functions or free functions.
```

Comment:
Actually that's not necessarily true - it depends on how we will implement `using` in the future. So maybe to clarify that this is about the type of a variable / expression and not about the function declaration itself.

Reply:
Hmm I am not sure how I would phrase that. Could you elaborate your example?

Reply:
true iff the function is called as arg1.fun(arg2, ..., argn). This is achieved through the "using for" directive.